sed/core/metadata.py

```diff
@@ -18,8 +20,31 @@ def __getitem__(self, val: Any) -> None:
         return self._m[val]
 
     def __repr__(self) -> str:
-        # TODO: #35 add pretty print, possibly to HTML
-        return str(self._m)
+        return yaml.dump(self._m, allow_unicode=True, default_flow_style=False)
+
+    def _format_attributes(self, attributes, indent=0):
+        html = ""
+        for key, value in attributes.items():
+            # Format key
+            formatted_key = key.replace("_", " ").title()
+            formatted_key = f"<b>{formatted_key}</b>"
```

Comment:
I'd suggest to remove this, or at least keep the real key in brackets or something. It makes parsing the dictionary based on its representation impossible or very difficult.

Reply:
I have put the key in [] now.